packages/venia-concept/src/components/MiniCart/miniCart.css

```diff
@@ -72,6 +72,33 @@
     margin-left: 0.5rem;
 }
 
+.focusItem {
+    display: block;
+    box-shadow: 0 1px rgb(var(--venia-border));
+    padding: 1rem;
+}
+
+.focusItem span {
```

Comment:
Try to avoid this at all costs, since it relies on the cascade. When we have extensibility, a component such as this will have no idea what `span` elements are below it in the tree; this kind of style rule would eliminate the limited benefits that CSS Modules gives us.